### PR TITLE
Get rid of depends on kubectl in kubeadm

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
Get rif of depends on kubectl in kubeadm
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/completion.go [hunk 1]
```diff
@@ -103,7 +102,7 @@ func GetSupportedShells() []string {
 func NewCmdCompletion(out io.Writer, boilerPlate string) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "completion SHELL",
-		Short:   i18n.T("Output shell completion code for the specified shell (bash or zsh)."),
+		Short:   kubeadmutil.T("Output shell completion code for the specified shell (bash or zsh)."),
```
Comment (stewart-yu) [inline cmd/kubeadm/app/cmd/completion.go]: Why not just grid of `i18n.T()` function, use `string` istead ?

**Review by fabriziopandini — CHANGES_REQUESTED** [review]
@hanxiaoshuai thanks for this PR!
I'm in favour of removing this dependency, because `i18n` is basically not used in kubeadm.
However, I would like to align the two commands using ``i18n` to all the other kubeadm commands using `string` instead of introducing a new external library.

**Review by luxas — COMMENTED** [review]
Thanks @hanxiaoshuai for this PR.
Actually, as we don't support translation at all in kubeadm (yet!) I'm fine with just removing the deps totally, not adding the new `T` func and just using plain strings in these two places.

**Comment by hanxiaoshuai** [comment]
@fabriziopandini @luxas Done

**Review by luxas — APPROVED** [review]
Thanks @hanxiaoshuai!
/lgtm
/approve

**Pull request merged**